frontend/src/components/hook-form/RHFAutoMultiSelect.tsx

```diff
@@ -54,14 +54,7 @@ export default function RHFAutoMultiSelect({ name, options = [], ...other }: Pro
             </li>
           )}
           renderInput={(params) => (
-            <TextField
-              {...field}
-              {...params}
-              inputRef={ref}
-              error={!!error}
-              helperText={error?.message}
-              {...other}
-            />
+            <TextField {...params} error={!!error} helperText={error?.message} {...other} />
```

Comment:
Just curious, what is the rationale behind removing this>

Reply:
removed {...field} cos idw to let the RHF control the manage the input and value for this field component

Reply:
I see